lerobot/configs/default.yaml

```diff
@@ -5,10 +5,17 @@ defaults:
 
 hydra:
   run:
+    # Set `dir` to where you would like to save all of the run outputs. If you run another training session
```

Comment:
FYI: I tried the other alternative as well: "the checkpoint configuration takes precedence". But I realised there would often be cases where one wants to stop a run and continue it with a tweak. For example, maybe they want to extend the number of training steps, or maybe they used the wrong env.task (this happened to me last night).

Reply:
Good idea. I've added `override_config_on_resume` (to be clear that it comes into play when using `resume`).